docs/platforms/python/integrations/redis/index.mdx

```diff
@@ -109,18 +109,27 @@ sentry_sdk.init(
     integrations=[
         RedisIntegration(
             max_data_size=None,
+            cache_prefixes=["mycache", "template.cache"],
         ),
     ],
 )
 ```
 You can pass the following keyword arguments to `RedisIntegration()`:
 
-- `max_data_size`:
+- `max_data_size`
 
 By default `RedisIntegration()` will trim data collected after `1024` characters. You can change this behavior with the `max_data_size` parameter:
 
-- When you set `max_data_size` to a value that evaluates to `False` (like `0` or `None`) no trimming will take place. The whole Redis command will be recorded.
-- Alternatively, you can set `max_data_size` to an integer to control how many characters should be collected.
+You can set `max_data_size` to an integer to control how many characters should be collected.
+
+When you set `max_data_size` to a value that evaluates to `False` (like `0` or `None`) no trimming will take place. The whole Redis command will be recorded.
+
+- `cache_prefixes`
+
+You can specify a list of prefixes to Redis keys to define a key space that should be considered as cache. Cache keys will show up in the [cache-monitoring dashboard](https://sentry.io/orgredirect/organizations/:orgslug/performance/cache/). Giving you more insight into your caching strategy.
```

Comment:
```suggestion
You can specify a list of prefixes to Redis keys to define a key space that should be considered as cache. Cache keys will show up in the [cache-monitoring dashboard](https://sentry.io/orgredirect/organizations/:orgslug/performance/cache/), giving you more insight into your caching strategy.
```

Reply:
Note: I believe this URL to the product will change for the June 5 launch since caches will live under "insights" instead of "performance." Let's remove the URL for now as we'll need to update it later.